pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -49,8 +48,10 @@ func deepEqualWithoutGeneration(actual *nodeInfoListItem, expected *framework.No
 	if expected != nil {
 		expected.Generation = 0
 	}
-	if actual != nil && !reflect.DeepEqual(actual.info, expected) {
-		return fmt.Errorf("got node info %s, want %s", actual.info, expected)
+	if actual != nil {
+		if diff := cmp.Diff(actual.info, expected, cmp.AllowUnexported(framework.NodeInfo{})); diff != "" {
+			return fmt.Errorf("got node info %s, want %s", actual.info, expected)
```

Comment:
Please show `diff` in the error message.

Reply:
Comment addressed. Please review the update.

Reply:
Comment addressed. Please review the update.